src/System.Management.Automation/engine/CommandCompletion/CompletionCompleters.cs

```diff
@@ -7080,6 +7080,16 @@ public object VisitParenExpression(ParenExpressionAst parenExpressionAst)
         {
```

Comment:
@rjmholt, your last commit had 1 failures in `PowerShell-CI-windows`
Verify PowerShell Runs.Verify Version 
```powershell
Expected regular expression '^7\.' to match $null, but it did not match.
at <ScriptBlock>, D:\_work\1\s\test\nanoserver\nanoserver.tests.ps1: line 16
16:         $version | Should -match '^7\.'
```